environment.yml

```diff
@@ -10,6 +10,7 @@ dependencies:
   - numpy=1.15.4
   - cftime=1.0.1
   - python-stratify=0.1
+  - conda-pack
```

Comment:
```suggestion
```

Reply:
Might make more sense to leave this out and keep it as part of a secondary requirements files (containing test packages:, sphinx/pytest/pylint as well?)